Library/Formula/libvisio.rb

```diff
@@ -15,10 +14,11 @@ class Libvisio < Formula
   depends_on "pkg-config" => :build
   depends_on "boost" => :build
   depends_on "cppunit" => :build
-  depends_on "libwpd"
-  depends_on "libwpg"
-  depends_on "icu4c"
   depends_on "librevenge"
+  depends_on "icu4c"
+  depends_on "libxml2"
+  depends_on "homebrew/dupes/gperf"
+  depends_on "homebrew/dupes/zlib"
```

Comment:
honestly, I don't know if using the dupes (gperf and zlib) are necessary for sure. What I do know is that they [are actual dependencies](https://wiki.documentfoundation.org/DLP/Libraries/libvisio#Building_the_code) and that from my experience using building an application using homebrew, Scribus, that I needed to use the zlib dupe. So I was playing it safe.